forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/main/java/com/forgerock/openbanking/aspsp/rs/api/payment/verifier/DetachedJwsVerifier.java

```diff
@@ -100,26 +103,30 @@ public void verifyDetachedJws(String detachedJws, OBVersion obVersion, HttpServl
         }
     }
 
-    private boolean isB64ClaimHeaderSetToFalse(String jwsDetachedSignature, String payload) throws OBErrorException, ParseException {
+
+    /**
+     * Returns true if the b64 header is true. This indicates that the base 64 encoded payload is included in the jws.
+     * @param jwsDetachedSignature
+     * @param payload
+     * @return
+     * @throws OBErrorException
+     * @throws ParseException
```

Comment:
Incomplete javadoc. Personally, I wouldn't bother on a private method as I'd hope it would be self-explanatory.